repo/packages/C/crate/0/marathon.json

```diff
@@ -0,0 +1,26 @@
+{
+    "id": "/crate/{{crate/cluster-name}}",
+    "instances": 1,
+    "cpus": 0.25,
+    "mem": 128,
+    "ports": [{{framework/api-port}}],
```

Comment:
instances, cpus, mem and ports all need to be configurable.

ex:  https://github.com/mesosphere/universe/blob/version-1.x/repo/packages/M/marathon/1/marathon.json#L3-L5